lms/djangoapps/instructor_task/api.py

```diff
@@ -14,16 +14,15 @@
 from bulk_email.models import CourseEmail
 from lms.djangoapps.certificates.models import CertificateGenerationHistory
 from lms.djangoapps.instructor_task.api_helper import (
-    check_arguments_for_rescoring,
     check_arguments_for_overriding,
+    check_arguments_for_rescoring,
     check_entrance_exam_problems_for_rescoring,
     encode_entrance_exam_and_student_input,
     encode_problem_and_student_input,
     submit_task
 )
 from lms.djangoapps.instructor_task.models import InstructorTask
 from lms.djangoapps.instructor_task.tasks import (
-    override_problem_score,
```

Comment:
Nit: why you move it?

Reply:
Import order fixed by using `isort` lib.